lib/rand.c

```diff
@@ -100,9 +100,9 @@ CURLcode Curl_win32_random(unsigned char *entropy, size_t length)
 }
 #endif
 
-#if !defined(USE_SSL) || defined(USE_RUSTLS)
+#if !defined(USE_SSL)
 /* ---- possibly non-cryptographic version following ---- */
-CURLcode Curl_weak_random(struct Curl_easy *data,
+static CURLcode Curl_weak_random(struct Curl_easy *data,
```

Comment:
This fixes the build:

```suggestion
static CURLcode weak_random(struct Curl_easy *data,
```

Reply:
:facepalm: Ah of course. I was trying to fix this the wrong way. Adjusting.

Reply:
Backed out my silly forward decl and updated with this fix. Thanks!